libraries/redcore/model/admin/base.php

```diff
@@ -160,6 +160,18 @@ protected function loadForm($name, $source = null, $options = array(), $clear =
 		RForm::addFormPath(JPATH_COMPONENT . '/models/forms');
 		RForm::addFieldPath(JPATH_COMPONENT . '/models/fields');
 
+		// Add form path from MVC override plugin
+		if (method_exists('RModelAdminBase', 'addComponentFormPath'))
```

Comment:
It exists in MVC override plugin here https://github.com/redCOMPONENT-COM/mvcOverride/blob/develop/core/model/modelform.php#L106
It need for can be possible replace xml files for components, which use redcore and override plugin.